test.fs

```diff
@@ -362,5 +362,69 @@ XC!+ XC!+? XC@+
 CHAR ( EKEY>XCHAR X-WIDTH XC-WIDTH [CHAR] ( 
 XHOLD
 -TRAILING-GARBAGE
+( Plugin Functionality Tests)
+
+\ --- Matchit
+
+\ TEST:    Matchit functionality for colon definitions.
+\ EXECUTE: With the cursor intially positioned on the ":" or ":NONAME" in the
+\   examples, execute the Vim % command repeatedly.
+\ RESULT:  The cursor should cycle between ":" or ":NONAME",
+\   "EXIT" and ";" ignoring those words contained in strings and
+\   comments.
+
+: MATCHIT ( -- )
+    S" skip these : :NONAME EXIT ;"
+    EXIT
+    ( and these : :NONAME EXIT ;)
+;
+
+:NONAME ( -- )
+    S" skip these : :NONAME EXIT ;"
+    EXIT
+    ( and these : :NONAME EXIT ;)
+;
+
+\ TEST:    Matchit functionality for conditional compilation
+\   words.
+\ EXECUTE: With the cursor intially positioned on the "[IF]" in the
+\   example, execute the Vim % command repeatedly.
+\ RESULT:  The cursor should cycle between "[IF]", "[ELSE]" and
+\   "[THEN]" ignoring those words contained in strings and
+\   comments.
+
+0 [IF]
+    S" skip these [ELSE] [THEN]"
```

Comment:
This test seems to fail for me. Do you get the same?

Reply:
Yes, this one probably depends on #25.  If you change it to `1 [IF]` it should work.